version.php

```diff
@@ -25,7 +25,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->component = 'mod_zoom';
-$plugin->version = 2020061200;
+$plugin->version = 0;
```

Comment:
Here's the original value:
```suggestion
$plugin->version = 2020061200;
```

Reply:
Damnit thought I’d done exactly that!